### PR TITLE
Add wg-container-identity

### PR DESCRIPTION
**Pull request by smarterclayton**
Needs generated contents

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -661,3 +661,23 @@ workinggroups:
...
+      company: Google
+    meetings:
+    - day: Tuesday
+      utc: "15:00"
+      frequency: bi-weekly (On demand)
+    meeting_url: TBD
+    meeting_archive_url: https://docs.google.com/document/d/1bCK-1_Zy2WfsrMBJkdaV72d2hidaxZBhS5YQHAgscPI/edit
+    contact:
+      slack: sig-auth
+      mailing_list: https://groups.google.com/forum/#!forum/kubernetes-sig-auth
```
Comment (jbeda) [inline sigs.yaml]: Working groups should have their own mailing lists and slack channels. They cut across SIGs.

Comment (smarterclayton) [inline sigs.yaml]: OK, these are placeholders until they are created.

**Comment by smarterclayton** [comment]
Updated with generated code and wg specific lists.

**Pull request merged**